stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Page.java

```diff
@@ -9,25 +9,27 @@
 
 package com.facebook.stetho.inspector.protocol.module;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-
 import android.content.Context;
+import com.facebook.stetho.common.LogUtil;
 import com.facebook.stetho.common.ProcessUtil;
+import com.facebook.stetho.inspector.domstorage.SharedPreferencesHelper;
 import com.facebook.stetho.inspector.jsonrpc.JsonRpcPeer;
 import com.facebook.stetho.inspector.jsonrpc.JsonRpcResult;
 import com.facebook.stetho.inspector.protocol.ChromeDevtoolsDomain;
 import com.facebook.stetho.inspector.protocol.ChromeDevtoolsMethod;
-import com.facebook.stetho.inspector.domstorage.SharedPreferencesHelper;
+import com.facebook.stetho.inspector.protocol.ScreencastDispatcher;
 import com.facebook.stetho.json.annotation.JsonProperty;
 import com.facebook.stetho.json.annotation.JsonValue;
-
 import org.json.JSONObject;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
 public class Page implements ChromeDevtoolsDomain {
   private final Context mContext;
+  private ScreencastDispatcher mScreencastDispatcher = new ScreencastDispatcher();
```

Comment:
You probably want this to not be constructed here but rather managed lazily when a caller actually asks for screen casting.  A subclassed `ChromePeerManager` may well be what you're missing...